esphome/components/sdm220m/sensor.py

```diff
@@ -0,0 +1,90 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import sensor, modbus
+from esphome.const import CONF_ACTIVE_POWER, CONF_APPARENT_POWER, \
+    CONF_CURRENT, CONF_EXPORT_ACTIVE_ENERGY, \
+    CONF_EXPORT_REACTIVE_ENERGY, CONF_FREQUENCY, CONF_ID, \
+    CONF_IMPORT_ACTIVE_ENERGY, CONF_IMPORT_REACTIVE_ENERGY, \
+    CONF_PHASE_ANGLE, CONF_POWER_FACTOR, CONF_REACTIVE_POWER, \
+    CONF_VOLTAGE, ICON_COUNTER, ICON_CURRENT_AC, ICON_FLASH, \
+    ICON_POWER, UNIT_AMPERE, UNIT_DEGREES, UNIT_EMPTY, UNIT_HERTZ, \
+    UNIT_VOLT, UNIT_VOLT_AMPS, UNIT_VOLT_AMPS_REACTIVE, \
+    UNIT_VOLT_AMPS_REACTIVE_HOURS, UNIT_WATT, UNIT_WATT_HOURS
+
+AUTO_LOAD = ['modbus']
```

Comment:
Please add yourself as a codeowner and then run `script/build_codeowners.py`

```suggestion
AUTO_LOAD = ['modbus']
CODEOWNERS = ['@polyfaces']
```